src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GoogleServicesDAO.scala

```diff
@@ -21,7 +23,9 @@ trait GoogleServicesDAO {
 
   def getMaximumAccessLevel(userId: String, workspaceId: String): Future[WorkspaceAccessLevel]
 
-  def getBucketName(workspaceId: String): String
+  def getBucketName(workspaceId: String) = s"rawls-${workspaceId}"
```

Comment:
Moved from `HTTPGoogleServicesDAO` because it does not query Google, and it the test version is identical.